apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -8,6 +8,99 @@ window.PLOrderBlocks = function (uuid, options) {
 
   let optionsElementId = '#order-blocks-options-' + uuid;
   let dropzoneElementId = '#order-blocks-dropzone-' + uuid;
+  let pressedKeys = {};
+
+  function initializeKeyboardHandling(optionsElementId, dropzoneElementId) {
+    function inDropzone(block) {
+      let parentArea = block.closest('.pl-order-blocks-connected-sortable');
+      return parentArea.classList.contains('dropzone');
+    }
+    function getIndentation(block) {
+      return Math.round(parseInt(block.style.marginLeft.replace('px', '') / TABWIDTH));
+    }
+    let fullContainer = document.querySelector('.pl-order-blocks-question-' + uuid);
+    let blocks = fullContainer.querySelectorAll('*.pl-order-block');
+
+    for (let block of blocks) {
+      block.addEventListener('click', () => {
+        block.focus();
+      });
+
+      block.addEventListener('blur', () => {
+        if (block.classList.contains('selected')) {
+          block.classList.remove('selected');
+        }
+      });
+
+      block.addEventListener('keydown', (ev) => {
+        pressedKeys[ev.key] = true;
+        if (!block.classList.contains('selected')) {
+          if (ev.key === 'Enter') {
+            block.classList.add('selected');
+          }
+        } else {
+          ev.preventDefault();
```

Comment:
Here we are overriding the behavior of the whole keyboard whenever a block is selected, we should only override the default behavior (by calling `ev.preventDefault()`) in the case that they use one of the keys we need to override.